Source/Core/VideoCommon/MainBase.cpp

```diff
@@ -45,11 +45,20 @@ void VideoBackendBase::Video_ExitLoop()
   s_FifoShuttingDown.Set();
 }
 
+void VideoBackendBase::Video_Cleanup()
+{
+  // First stop any framedumping, which might need to dump the last xfb frame. This process
+  // can require additional graphics sub-systems so it needs to be done first
+  g_renderer->ExitFramedumping();
+
+  Video_CleanupImpl();
```

Comment:
IMO it's better to follow the naming convention already in place here of the "Shared" suffix, rather than having one method which uses "Impl". It's not really an "implementation" as such, anyway.

Reply:
I don't really mind, was just trying to come up with a way to differentiate the shared logic with the backend specific logic.  I can change this to Video_CleanupBackend() or change the other one to Video_CleanupShared() and change this to Video_Cleanup()